src/transformers/__init__.py

```diff
@@ -48,6 +48,7 @@
 
 # Base objects, independent of any specific backend
 _import_structure = {
+    "adapters": [],
```

Comment:
Not sure we will have that many adapters to justify a new folder. How about an `integrations` folder where we would have PEFT, maybe move bitsandbytes to it and deepspeed, and more generally put submodules likes to integrations with other libs?

Reply:
Agreed, what about moving all other folders in a follow up PR?

Reply:
Needs to be done within the same release cycle so good for me as long as this is merged shortly after the next release branch cut (so that the PR moving everything is done before the release after)

Reply:
sounds great!

Reply:
decided to name the folder `lib_integrations` as a file `integrations.py` already exists (which created conflicts), let me know if you have a better name in mind (I can also rename `integrations.py` file)